research/src/components/breadcrumb-anatomy.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+import React from 'react'
+import './anatomy.css'
+import './breadcrumb-anatomy.css'
+
+const BreadcrumbAnatomy = () => {
+  return (
+    <div class="component-anatomy-wrapper">
+      <input type="checkbox" id="show-slots" />
+      <label for="show-slots"> Show slots</label>
+      <div class="component-anatomy">
+        <host name="breadcrumb">
```

Comment:
As we discussed on the call - here is the anatomy I'm proposing so that we can limit the explanations later since you're not actually proposing new components nor elements.

```
<host name="bread-crumb">
     <slot name="breadcrumb-container">
        <ol>
            <li part="breadcrumb-item"></li>
            <li part="breadcrumb-divider" aria-hidden="true"></li>
        </ol>
     </slot>
</host>
```

cc: @chrisdholt

Reply:
That's a little confusing, has the anatomy for should just represent where we would place the "known" components, like the `Table` anatomy that shows where we place `table-row`, `table-header`, `table-pagination` etc... If I naively apply the proposed I get this rendered:

<img width="924" alt="Screen Shot 2020-09-23 at 05 01 28" src="https://user-images.githubusercontent.com/8545105/93961387-e0950280-fd59-11ea-9bff-60b871538c9e.png">

Trying to find a common ground I am applying this:

```jsx
        <host name="breadcrumb" role="nave">
          <slot name="breadcrumb-container">
            <part name="ol">
              <part name="li">item</part>
              <part name="li" aria-hidden="true">
                divider
              </part>
            </part>
          </slot>
        </host>
```

<img width="925" alt="Screen Shot 2020-09-23 at 05 02 28" src="https://user-images.githubusercontent.com/8545105/93961889-015d5800-fd5a-11ea-87dd-8b3f5ff77826.png">

Reply:
Also edited the structure section descriptions

Reply:
@assuncaocharles that's precisley what I meant by the HTML I wrote. Only modification is to have this:


`<part name="li [part=item]">` and the same for divider. You can leave the content of item and divider there

Reply:
I've got some concerns on the above and would like to further discuss before this gets approved. I think I'm more closely aligned to an *end result* of what Charles initially had. I brought this to some folks on our team and the following things came up:

- We aren't sure that the separator should be a list item
- I'm not sure I would expect the list item (li) itself to be the actionable bit. 
    - What affordance does a non-sighted user have to know that's actionable?
    - Slotted content for an item makes more sense to me here, but there isn't an "item" concept so that doesn't seem possible. That would allow me to reuse the anchor and get it's native behaviors and semantics, which is valid. Consider that I want to open these links in a new tab, or a new window because it's a form and I don't want to navigate you away from the page.
    - It seems like we are convoluting multiple semantic roles with the current implementation. `role="listitem"` + "role="link".

As it stands now, I'd be a voice of dissent in moving towards this implementation.

Reply:
Hey @chrisdholt 

    1. I think divider should be list item, it's the common across most of the other libs and even aria recommendation. Of course the implementation can change and like in Fabric it can be rendered inside of `Item` for several reasons. 
    2. I don't think the item ( `li` ) should be actionable at all also, it can contain anchor or texts inside.

Reply:
@gregwhitworth did your last update, this is what we have:

<img width="920" alt="Screen Shot 2020-09-24 at 07 53 40" src="https://user-images.githubusercontent.com/8545105/94106208-3c838800-fe3b-11ea-971e-6b75a124e0d0.png">

Reply:
@chrisdholt can you review please